packages/plugins/AddressSearch/src/components/Results.vue

```diff
@@ -100,6 +125,18 @@ export default Vue.extend({
     },
   },
   watch: {
+    featuresAvailable(): void {
+      this.$nextTick(() => {
+        const firstElement =
+          // @ts-expect-error | Type conversion is fine here as the querySelector method is monkeyPatched in core/createMap
+          (document.querySelector('[data-app]') as ShadowRoot).getElementById(
+            'polar-plugin-address-search-results-feature-0-0'
+          )
+        if (firstElement) {
+          firstElement.focus()
+        }
```

Comment:
Duplicate implementation to `focusFirstResult`.

Reply:
Deduplicated 2e873110